ArduPlane/mode_rtl.cpp

```diff
@@ -21,17 +21,17 @@ void ModeRTL::update()
 
     if (plane.g2.rtl_climb_min > 0) {
         /*
-          when RTL first starts limit bank angle to LEVEL_ROLL_LIMIT
-          until we have climbed by RTL_CLIMB_MIN meters
+          when RTL first starts limit bank angle until we have climbed
+          by RTL_CLIMB_MIN meters above RTL start or ALT_HOLD_RTL,
+          whichever is highest.
          */
-        if (!plane.rtl.done_climb && (plane.current_loc.alt - plane.prev_WP_loc.alt)*0.01 > plane.g2.rtl_climb_min) {
+        if (!plane.rtl.done_climb && (plane.current_loc.alt > MAX(plane.next_WP_loc.alt,plane.prev_WP_loc.alt))*0.01 + plane.g2.rtl_climb_min) {
             plane.prev_WP_loc = plane.current_loc;
             plane.setup_glide_slope();
             plane.rtl.done_climb = true;
         }
         if (!plane.rtl.done_climb) {
-            plane.roll_limit_cd = MIN(plane.roll_limit_cd, plane.g.level_roll_limit*100);
-            plane.nav_roll_cd = constrain_int32(plane.nav_roll_cd, -plane.roll_limit_cd, plane.roll_limit_cd);
+            plane.nav_roll_cd = constrain_int32(plane.nav_roll_cd, -1, 1);
```

Comment:
I think its better to leave with the level roll limit, you can then just change the param to 0 if you want 0. 

This current change might as well be 
```suggestion
            plane.nav_roll_cd = 0.0f;
```

Reply:
This is true, what I'm unsure of is this part of the docs:

> Setting this to zero will completely disable heading hold on auto takeoff and final landing approach.

Given that, I wasn't sure what the impact of setting this to zero would be to RTL.